string pcd_path
float32 z_value
---
bool status